File: 프로그래머스/lv0/120887. k의 개수/k의 개수.js
function solution(i, j, k) {
    let result = 0;

    for (let n = i; n <= j; n++) {
        if (n.toString().includes(k.toString()))
          result += n
            .toString()
            .split("")
            .filter((str) => str === k.toString()).length;
    }
    return result;
}

File: 프로그래머스/lv0/120887. k의 개수/k의 개수.java
import java.util.Arrays;
import java.util.stream.IntStream;
class Solution {
    public int solution(int i, int j, int k) {
        return IntStream.rangeClosed( i, j )
						.mapToObj( String :: valueOf )
						.flatMap( str -> Arrays.stream( str.split( "" ) ) )
						.filter( ch -> ch.equals( String.valueOf( k ) ) )
						.toArray().length;
    }
}